blocks/library/gallery/index.js

```diff
@@ -121,16 +124,24 @@ registerBlockType( 'core/gallery', {
 			controls,
 			focus && images.length > 1 && (
 				<InspectorControls key="inspector">
+					<p className="editor-block-inspector__description">Image galleries are a great way to share groups of pictures on your site.</p>
```

Comment:
just mind note: we should create a component for this `<BlockDescription>` to avoid copying and relying on classes.